devops/helm/clamav/main/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-same-namespace
```

Comment:
is it not going to affect our existing NetworkPolicies named `allow-same-namespace`?

Reply:
These things can be removed safeside, i just copy pasted the whole helm/main folder. This is not applied during the helm install/upgrade/uninstall of the clamAV server. So this should not affect our network configuration.